provider/consumer.py

```diff
@@ -313,6 +313,7 @@ def __fireTrigger(self, messages):
                     # service was unavailable (503), an internal server error occurred (500), the request timed out
                     # (408), or the request was throttled (429).
                     if status_code == 200:
+                        logging.info("[{}] Fired trigger with activation {}".format(self.trigger, response.json()['activationId']))
```

Comment:
Should we also log the activation when the trigger was not successfully fired and an activationId exists in the response?